agent/app/agent.go

```diff
@@ -402,6 +404,10 @@ func (agent *ecsAgent) registerContainerInstance(
 			seelog.Critical("Instance registration attempt with an invalid attribute")
 			return err
 		}
+		awsErr, ok := err.(awserr.Error)
+		if ok && awsErr.Code() == ecs_service.ErrCodeInvalidParameterException {
+			return err
```

Comment:
Can you also add a log line here? Similar to the `"invalid attribute"` one above?

Reply:
done